js/about/brave.js

```diff
@@ -43,7 +43,7 @@ class AboutBrave extends React.Component {
       <div className='siteDetailsPageContent aboutAbout'>
         <div className='title'>
           <span className='sectionTitle' data-l10n-id='versionInformation' />
-          <span className='fa fa-clipboard' title='Copy password to clipboard' onClick={this.onCopy} />
```

Comment:
Two things ( cc: @Liunkae ):
1. We're missing `data-l10n-id` for this item. We need it to look like this:
`<span data-l10n-id='copyToClipboard' className='fa fa-clipboard' onClick={this.onCopy} />`
2. When you look at about-brave.html, you'll see that for localization it uses `app.properties`.  Inside this file, you'll need to be having something like this: `copyToClipboard.title=Copy to clipboard`